bin/sl-pm.js

```diff
@@ -34,7 +34,8 @@ function printHelp($0, prn) {
 
 var argv = process.argv;
 var $0 = process.env.CMD ? process.env.CMD : path.basename(argv[1]);
-var parser = new Parser([
+var parser = new Parser(
+  [
```

Comment:
This doesn't look right, we'd always write (I think, I would anyhow, I think you would too):
```
var parser = new Parser({
  a: 1,
  b: 2,
});
```
so wouldn't we always write
```
var parser = new Parser([
  1,
  2,
]);
```
?

What's happening here? Is it because there are two args, so eslint wants them to be both on individual lines? Or is array literals being treated differently from object literals?